app/components/collections/detail_component.html.erb

```diff
@@ -1,8 +1,11 @@
 <table class="table table-sm mb-5 caption-header">
   <caption>Details
-    <%= link_to edit_collection_version_path(collection_version), aria: { label: 'Edit details' } do %>
-      <span class="fa-solid fa-pencil-alt edit" aria-hidden="true"></span>
-    <% end %>
+    <%= helpers.turbo_frame_tag dom_id(collection_version, :edit),
+                                src: edit_link_collection_version_path(
+                                  collection_version,
+                                  label: 'Edit details'
+                                ),
+                                target: '_top' %>
```

Comment:
Wrap this in a turbo frame else we get warden errors when the turbo stream renders it server-side.